tests/unit/components/sl-calendar-test.js

```diff
@@ -270,6 +254,13 @@ test( 'Months for year view are generated validly', function( assert ) {
     );
 });
 
+skip( 'monthsInYearView - active month set correctly', function() {
+    /* Expand preceeding test to also check that the active month is set correctly.
```

Comment:
Reference the name of the test rather than "preceding" as the code may get moved around before this code is worked on.  Even better is to create an issue referring to the refactor desired to be applied to the desired test.